model-events/src/main/java/de/focus_shift/urlaubsverwaltung/extension/api/application/ApplicationAllowedEventDTO.java

```diff
@@ -0,0 +1,41 @@
+package de.focus_shift.urlaubsverwaltung.extension.api.application;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.Set;
+import java.util.UUID;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class ApplicationAllowedEventDTO {
+  @NonNull private UUID id;
+
+  @NonNull private Instant createdAt;
```

Comment:
Maybe allowedAt or is it the date when the application was created?

Reply:
neither one of them:
* id: UUID of the event
* createdAt: Timestamp when the event was created
=> both properties have no business relation to the application by itself, those fields are just in the realm of the event.